Game.java

```diff
@@ -23,37 +23,34 @@ public void switchTurn() {
         playerDeactive = newActivePlayer;
     }
 
-    public void print(String text) {
-        System.out.println(text);
-    }
-
-    public String showBoard(Board board) {
-        print("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
-        return String.format("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|");
-    }
-
     public Board playerMakesMove() {
         int position = playerActive.nextMove(board);
         if (board.availablePosition(position - 1)) {
             board = board.playerMakesMove(playerActive.getMark(), position - 1);
             switchTurn();
-            print("-----------------");
-            print("Its your turn " + playerActive.getMark() + "!");
-            print("-----------------");
-            showBoard(board);
-            print("-----------------");
+            gameType.drawTurn(this);
+            gameType.drawBoard(board);
             return board;
         }
-        print(position + " is not a valid position!");
+        gameType.invalidMove();
         return board;
     }
 
+    public Player playerActive() {
```

Comment:
are these used outside of Game?  I'd hope not, so the getters are possibly unnecessary..